src/components/custom-aggrid/custom-aggrid-header.jsx

```diff
@@ -74,6 +74,10 @@ const CustomHeaderComponent = ({
         isDuration, // if the value is a duration, we need to handle that special case, because it's a number filter but with text input
     } = filterParams;
 
+    const isMultipleSelection = !(
```

Comment:
this creates a regression, we can not filter by multiple selection for enums with two options